packages/components/src/toolbar-button/index.js

```diff
@@ -21,6 +21,9 @@ function ToolbarButton( {
 	className,
 	extraProps,
 	children,
+	title,
+	isActive,
```

Comment:
Aside: So much legacy here :( Ideally, we should be promoting `label` and `isPressed` to make it easier for developers to use API. At the same time, `disabled` in the `Button` should be really `isDisabled` 😃 
Nothing actionable for now, I just wanted to complain a bit 🙃

Reply:
I think we should eventually deprecate those props and make the API the same as on `Button`. I chose to add those props here because there is already a ton of code using `ToolbarButton` with them, and the migration would be more painful.

Reply:
Agreed 💯